charts/k8ssandra-operator/Chart.yaml

```diff
@@ -9,7 +9,7 @@ dependencies:
     version: 0.29.0
     repository: https://helm.k8ssandra.io
   - name: cass-operator
-    version: 0.43.0
+    version: 0.44.0
```

Comment:
**Question:** shouldn't we go straight to `0.44.1` now that it's out?

Reply:
Sure.